### ISSUE TITLE
What version of jdk, ndk and gradle did you use for this project?

### ISSUE THREAD

**Issue by dothedotheg** [biosignalsplux/unity-android-sample#4]
Hello, I am trying to build this unity project.
However I failed to build this project. 
I guess this might be due to the miss matching of the jdk, ndk and the gradle I use.
But I couldn't find the information about it. Could you let me know what version you used?
If this is not the problem of version do you know what is causing the error?

I am using the 
SDK Platform 27 [Android 8.1]
Unity 2022.1.16f

As the document recommended.

![error00](https://user-images.githubusercontent.com/54068517/229627711-171f62cb-fdbf-47b1-b8db-9ada75444aa4.PNG)
![error01](https://user-images.githubusercontent.com/54068517/229627720-2e1dd9ae-be4a-4a81-b71b-e71a0739c8b4.PNG)
![error02](https://user-images.githubusercontent.com/54068517/229627727-ba0d1fe4-38fd-4947-8c0c-1b06d9076fa0.PNG)
![error03](https://user-images.githubusercontent.com/54068517/229627737-7a366538-d288-42d7-a03b-443cdf96ca0a.PNG)
![error04](https://user-images.githubusercontent.com/54068517/229627745-185c9310-0af1-411e-b9ee-4ec55563003e.PNG)

**Comment by dothedotheg** [comment]
![error01](https://user-images.githubusercontent.com/54068517/229627643-e2335b9b-86dd-45f5-9eee-079e9cb00f22.PNG)

**Comment by v-gramos** [comment]
Hello @dothedotheg,

Thank you for your contact! 
We tried to replicate your problem but were unsuccessful. While searching for different possible origins for this problem we came across the following [post](https://forum.unity.com/threads/gradle-error-problem-with-gradle-build.538320/).

In order for us to help we ask you to answer the following questions.

1. Can you please send us a screenshoot of your build settings?

![Screenshot 2023-04-04 at 17 16 20](https://user-images.githubusercontent.com/61143066/229854091-1796164a-cf2f-45cc-9068-7fb3ef406132.png)

2. Do you have some keystone manager defined?

![Screenshot 2023-04-04 at 17 17 40](https://user-images.githubusercontent.com/61143066/229854390-63baa2be-4d22-4727-b7a0-06ba13e06b6f.png)

3. Is the F Drive your system disk?


Best regards,
v-gramos

**Comment by GRamosPlux** [comment]
The issue seems to have been overtaken by @dothedotheg taking into consideration the developments reported on **Issue #5**

**Comment by dothedotheg** [comment]
Hello,
Thank you for your response.
I solved the problem by deleting Java and reinstalling the latest version.
And now there is no error at all with the latest unity version.
What I learned while working on this project is that the files uploaded to
GitHub are fine, but they cannot be used in VR. I tried to use Meta Quest 2
by connecting it to the Bioflux Signal Solo, but I couldn't succeed because
Meta blocked unauthorized Bluetooth devices.

Thank you so much for caring and managing the github files.
All the best,
Hyun

2023년 4월 12일 (수) 오전 2:07, GRamosPlux ***@***.***>님이 작성:
